changes/2020-06-04_how-to-fail-with-keyrings/change.md

```diff
@@ -108,37 +108,43 @@ the security properties of each keyring.
 
 Clearly defining what causes keyrings to fail
 makes it simpler for users to predict failing behavior
-based on either environment
-and across implementations.
+in different environments and implementations.
 
 ## Guide-level Explanation
 
-When a keyring succeeds on encryption,
-it creates an artifact: one or more encrypted data keys.
+On encryption,
+a keyring creates an artifact:
+one or more encrypted data keys.
```

Comment:
See #158. There is some cross pollination of words that might be good to have.
Not necessarily something to block on.